src/views/Main/Provider/SettingsProvider.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+
+import withEnv from '../../../config/withEnv';
+
+import { contextSettings } from './context';
+
+const { Provider } = contextSettings;
+const defaultSettings = {
+  favicon: '/favicon.png',
+  title: 'terra-visu',
+  theme: {
+    logo: '/images/terravisu-logo.png',
+    style: [],
+  },
+};
+const getSettings =  async SETTINGS => {
```

Comment:
Why `SETTINGS` in uppercase here ?
also naming suggestion `settings_path`

Reply:
done